package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@moxystudio/create-next-routes",
+  "version": "1.0.0",
+  "description": "A CLI to manage Next.js routes",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest --coverage",
+    "prerelease": "npm t && npm run lint",
+    "release": "standard-version",
+    "postrelease": "git push --follow-tags origin master && npm publish"
+  },
+  "bin": {
+    "@moxystudio/create-next-routes": "bin/create-next-routes",
+    "create-next-routes": "bin/create-next-routes"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
In my opinion, I'll rather have to run `npm publish --access public` the first time and not pollute the `package.json`.